### ISSUE TITLE
Auto upload issue

### ISSUE THREAD

**Issue by kannansofts** [nextcloud/android#7755]
### Steps to reproduce
1. 
2. 
3. 

### Expected behaviour
- Tell us what should happen
If enable auto upload it should upload selected folder automatically.
### Actual behaviour
- Tell us what happens
Once configuration is saved same is not reflecting. We may need to remove the account from the app and again configure.
### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version: 10

Device model: OnePlus 6

Stock or customized system: Stock

Nextcloud app version: 3.14.1

Nextcloud server version: 20.0.4

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by AndyScherzinger** [comment]
Hi @kannansofts, do you mean any file or just newly created ones after activation of auto upload for a folder. In case if existing ones, did you activate this on the settings screen?
As for the account, do you suggest it is working if you remove and re-add the Nextcloud account and re-do the auto upload config?

**Comment by 1989gironimo** [comment]
Had the same issue, after readding the account the files get detected and uploaded again.
App version 3.15.0 on GrapheneOS (Android 11, newest patchlevel) without Play Services and Server version 20.0.7

**Comment by tobiasKaminsky** [comment]
> after readding the account the files get detected and uploaded again.

Sorry, I fail to understand it.
Is this the desired behavior or something you do not want?

**Comment by 1989gironimo** [comment]
Maybe I wasn't clear enough. 
I had to delete the app data via the settings, to reset the app basically. 
Afterwards I added the account again (and all the settings including the auto upload folder pairs) and checked if the auto upload is working again. 
If this is a onetime thing I don't mind, but this should be documented for those still having the issue. 

(So I think it is not a very convenient way to solve this, thus not desired )
